tests/env.sh

```diff
@@ -0,0 +1,6 @@
+export TF_ACC=true
```

Comment:
You shouldn't need this here if you are running `make testacc`

Reply:
@paultyng Indeed, but when I run some tests manually during development I use to `source tests/env.sh` then I'm able to run `go test ...` to target specific tests.

Reply:
I typically just do:

```shell
TESTARGS="-cover -run 'MyTest'" make testacc
```

But feel free to merge if you prefer your style.